### PR TITLE
Fixes #8 - compare test nmse of several approaches end to end  

### LINKED ISSUE #8: [Jaime] improve model-based prediction
In eth-predict2.md, improve model-based prediction. 

- [ ] write the code that merges several existing datasets, such that a single model can be trained. Put it in the README
- [ ] train a simple model on this. Update README
- [ ] try various tactics to reduce nmse: different feature vectors, different model parameters, different modeling algorithms, .. . Document results, possibilities are: in blog (see separate issue), same README, separate READMEs, separate py scripts in this repo

Parent epic: https://github.com/oceanprotocol/predict-eth/issues/3

### PR DESCRIPTION
**Pull request by jfdelgad**
Fixes #8  .

Changes proposed in this PR:

-  Add a readme describing comparison of 4 different models (linear regression, random forest, SVR, NN) for direct prediction of 1- 12 hours ahead of the current time
- 
-

### CODE REVIEW

File: examples/model_example_direct_prediction.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+# normalized mean-squared-error
+def nmse(y,yhat):
+    return np.sum(np.square(y - yhat))/np.sum(np.square(y))
+
+
+# regression using a base estimator and RegressionChain
+def fit_and_predict_reg(base_model,x_train,y_train,x_test,y_test):
+  chain = RegressorChain(base_estimator=base_model).fit(x_train, y_train)  
+  predicted = chain.predict(x_test)
+  return nmse(y_test,predicted)
```
Comment (trentmc) [inline examples/model_example_direct_prediction.md:74]: As discussed: please rename from "predicted" to "yhat_test"

**Review by trentmc — CHANGES_REQUESTED** [review]

File: examples/model_example_direct_prediction.md (hunk 1 above)
Comment (jfdelgad) [reply examples/model_example_direct_prediction.md]: Done

File: examples/model_example_direct_prediction.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  x = Dense(128,activation='relu')(inputs)
+  x = Dense(64,activation='relu')(x)
+  outputs = Dense(12,activation='linear')(x)
+  model = Model(inputs=inputs,outputs=outputs)
+  # set compiling parameters
+  model.compile( optimizer="adam", loss='mean_absolute_percentage_error',metrics=[])
+  # Fit model
+  model.fit(x_train,y_train,batch_size=100, epochs=500, validation_split=0.1, verbose=0)
+  # predict
+  predicted = model.predict(x_test)
```
[the remark this answers is not in the document]
Comment (jfdelgad) [reply examples/model_example_direct_prediction.md]: Done

File: ideas/modeling.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Static models
 Dynamic models (directly for time series):
 
 - Predict ETH challenge 1 winning algorithm
-- Recurrent network
+- [Model comparison for direct prediction 1-12 hours](../examples/model_example_direct_prediction.md)
```
[the remark this answers is not in the document]
Comment (jfdelgad) [reply ideas/modeling.md]: Done

File: README.md [hunk 4]
```diff
@@ -31,6 +31,7 @@ Here are ideas to get even more accurate results.
 - [More data sources](ideas/data-sources.md)
 - [Modeling approaches](ideas/modeling.md)
 - [Articles](ideas/articles.md) on predicting ETH, etc
+- [Model comparison for direct prediction 1-12 hours](../examples/model_example_direct_prediction.md)
```
Comment (trentmc) [inline README.md:34]: Please put this in the end-to-end section.

Then, approved! (Approving now, please don't wait on me after your final tweak.)

**Review by trentmc — APPROVED** [review]

**Pull request merged**